dianna/visualization/timeseries.py

```diff
@@ -11,13 +13,12 @@ def plot_timeseries(
     x: np.ndarray,
     y: np.ndarray,
     segments: List[Dict[str, Any]],
-    xlabel: str = 'x',
-    ylabel: str = 'y',
+    x_label: str = 't',
+    y_label: Union[str, Iterable[str]] = None,
     cmap: Optional[str] = None,
     show_plot: bool = False,
     output_filename: Optional[str] = None,
-    ax: Optional[plt.Axes] = None,
-) -> plt.Axes:
+) -> plt.Figure:
     """Plot timeseries with segments highlighted.
 
     Args:
```

Comment:
I think it is worth spelling out the expected shapes for the monovariabe/multivariate data here. This function expects channels to be on the first dimension. This may not be obvious to someone used to working with columnar data like myself, who will expect channels to be on the 2nd dimension.

Reply:
I added it in the docstring for y